extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -439,6 +445,9 @@ export class AmpA4A extends AMP.BaseElement {
       }
     }
 
+    this.detectedFont_ =
```

Comment:
Quoting @cramforce here: "This must be called directly in `onLayoutMeasure`, not in any of the callbacks scheduled from there."

We could override onLayoutMeasure in adsense-impl, make a call to super, then get the font.